service-tests/suggest.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'suggest', title: 'suggest', pathPrefix: '/$suggest' });
+module.exports = t;
+
+
+t.create('issues')
+  .get('/v1?url=' + encodeURIComponent('https://github.com/atom/atom'))
+  .addHeader('origin', 'https://shields.io')
+  .expectJSON('badges.?', {
+    name: 'GitHub issues',
+    link: 'https://github.com/atom/atom/issues',
+    badge: 'https://img.shields.io/github/issues/atom/atom.svg'
+  });
+
+t.create('forks')
+  .get('/v1?url=' + encodeURIComponent('https://github.com/atom/atom'))
+  .addHeader('origin', 'https://shields.io')
+  .expectJSON('badges.?', {
+    name: 'GitHub forks',
+    link: 'https://github.com/atom/atom/network',
+    badge: 'https://img.shields.io/github/forks/atom/atom.svg'
+  });
```

Comment:
Would it work to combine all of these into a single `t.create()`?

```js
t.create('issues')
  .get('/v1?url=' + encodeURIComponent('https://github.com/atom/atom'))
  .addHeader('origin', 'https://shields.io')
  .expectJson(...)
  .expectJson(...)
  .expectJson(...)
```

Reply:
I've chained issues, forks, stars and twitter.